### DISCUSSION TITLE
How to show all columns in editor

### DISCUSSION THREAD

**Discussion by mgdm187** [serenity-is/Serenity#5956] [Help] [answered]
Hi,
I have a dialog and new editor to add new data. But my problem is that this editor doesn't show all the columns I have. For example, I need to show column 'TrosakNaziv' and I included that but it doesn't show it on editor.
How could I show that column in editor?
![editor](https://user-images.githubusercontent.com/63368621/132000231-9948503e-af6f-4c4b-85ef-5c5b51388cba.png)

In my row I have this code for editor:
`[MasterDetailRelation(foreignKey: "AktivnostId", IncludeColumns = "TrosakId, TrosakNaziv, Napomena, Iznos, MjesecPlana")]
        [NotMapped, DisplayName("Planirani troškovi")]
        public List<MktPlaniranjeRow> PlaniranjeTroskova
        {
            get { return Fields.PlaniranjeTroskova[this]; }
            set { Fields.PlaniranjeTroskova[this] = value; }
        }`

BR

**Comment by mgdm187** [comment, marked as answer]
I found the answer to that.
I copied this code:
`protected getPersistanceStorage(): Serenity.SettingStorage {
            return new Common.UserPreferenceStorage();
        }`

in myEditor.ts and it works.

**Comment by luongnv6** [comment]
use  MinSelectLevel(SelectLevel.List) for field:
```C#
        [Expression("T0.Quantity"), MinSelectLevel(SelectLevel.List)]
        public Decimal? Quantity
        {
            get => fields.Quantity[this];
            set => fields.Quantity[this] = value;
        }
